modules/primo/src/Primo/Ting/Object.php

```diff
@@ -0,0 +1,258 @@
+   */
+  public function getSourceId() {
+    return $this->document->getSourceRecordId();
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function isLocal() {
+    // TODO: Implement isLocal() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getOwnerId() {
+    // TODO: Implement getOwnerId() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getTitle() {
+    return $this->document->getTitle();
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getShortTitle() {
+    return $this->document->getBriefTitle();
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getDescription() {
+    return $this->document->getDescription();
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getRelations() {
+    // TODO: Implement getRelations() method.
+  }
+
+
+  /**
+   * @inheritDoc
+   */
+  public function getAge() {
+    // TODO: Implement getAge() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getAudience() {
+    // TODO: Implement getAudience() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getExtent() {
+    return $this->document->getDisplayFormat();
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getFormat() {
+    // TODO: Implement getFormat() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getGenere() {
+    // TODO: Implement getGenere() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getIsbn() {
+    // A Primo record may contain multiple ISBNs - ISBN-10 and ISBN-13.
+    // Ding2 only works with a single ISBN value for an object and we prefer
+    // ISBN-13 so sort values by length and return the longest.
+    $isbns = $this->document->getIsbns();
+    usort($isbns, function($a, $b) {
+      return strlen($b) - strlen($a);
+    });
+    return reset($isbns);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getMusician() {
+    // TODO: Implement getMusician() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getPegi() {
+    // TODO: Implement getPegi() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getPublisher() {
+    return $this->document->getPublisher();
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getReferenced() {
+    // TODO: Implement getReferenced() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getReplacedBy() {
+    // TODO: Implement getReplacedBy() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getReplaces() {
+    // TODO: Implement getReplaces() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getRights() {
+    // TODO: Implement getRights() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getSeriesDescription() {
+    // TODO: Implement getSeriesDescription() method.
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public function getSource() {
+    $source = $this->document->getLocalDisplayField(6);
+    return str_replace('Á frummáli: ', '', $source);
```

Comment:
I guess `Á frummáli: ` is something in Icelandic indicating a text related to showing the source.
A comment would still be nice.